native/src/utils/renderHtml.ts

```diff
@@ -81,22 +79,41 @@ const renderJS = (cacheDictionary: Record<string, string>) => `
         return window.setTimeout(adjustHeight, 100);
       }
 
-      var height = container.getBoundingClientRect().height - 2
+      const height = container.getBoundingClientRect().height - 2
       window.ReactNativeWebView.postMessage(JSON.stringify({ type: '${HEIGHT_MESSAGE_TYPE}', height: height }));
       container.setAttribute('style', '');
     }
 
     window.addEventListener('load', adjustHeight);
     window.addEventListener('resize', adjustHeight);
-    var details = document.querySelectorAll("details")
+    const details = document.querySelectorAll("details")
     details.forEach(detail => detail.addEventListener("toggle", adjustHeight))
   })();
+
+  (function handleIframes() {
+    const iframes = document.querySelectorAll('iframe')
+    const allowedIframeSources = ${JSON.stringify(allowedIframeSources)}
+
+    iframes.forEach((el) => {
```

Comment:
Please use speaking names:
```suggestion
    iframes.forEach((iframe) => {
```